server/controllers/connectedAccounts.js

```diff
@@ -47,7 +48,25 @@ module.exports = (app) => {
       } else {
         return next(new errors.BadRequest('Github authorization failed'));
       }
-    }
+    },
 
+    fetchAllRepositories: (req, res, next) => {
+      const payload = req.jwtPayload;
+      ConnectedAccount
+      .findOne({where: {provider: 'github', id: payload.connectedAccountId}})
```

Comment:
Do we need a `provider` attribute in this clause? Shouldn't `id` identify it precisely?